### DISCUSSION TITLE
Declaring multiple licenses?

### DISCUSSION THREAD

**Discussion by bzg** [publiccodeyml/publiccode.yml#159] [Q&A]
Is it possible to declare multiple licenses in a `publiccode.yml` file?

https://reuse.software suggests to add multiple licenses in a dedicated `LICENSES` directory, which seems fine.

AFAIK neither GitHub nor GitLab detect multiples licenses in such a `LICENSES` directy (see the [example here](https://github.com/fsfe/reuse-example), where no license is indicated).

You can declare multiple licenses in a `codemeta.json` declaration, it would be good to be able to do so in a `publiccode.yml` declaration too. 

Thanks!

**Comment by bfabio** [comment]
@bzg good question :thinking: 

`legal/license` holds an SPDX identifier, so "`AGPL-3.0-or-later AND Apache-2.0 AND MIT`" is a valid value for `legal/license`, do you think that would be good enough?

**Comment by bzg** [comment]
Well, is it `AND` or `OR`? In general, I expect `OR` to be more accurate, when the author of a software wants to allow users to pick up one (and only one) license.

I was expecting we could simply have multiple yaml entries for the `license` field (hence avoiding to question whether the entries in this field are separated by `AND` or `OR`, which should be easily decided by reading the `README`): would that be possible?  The advantage is that we would parse the `license` field with no additional parsing trick.

**Reply by bfabio** [reply]
> I was expecting we could simply have multiple yaml entries for the `license` field (hence avoiding to question whether the entries in this field are separated by `AND` or `OR`, which should be easily decided by reading the `README`): would that be possible? The advantage is that we would parse the `license` field with no additional parsing trick.

Personally I think that it would make a lot of sense
